src/components/header.jsx

```diff
@@ -11,9 +15,16 @@ const HeaderLink = ({ link }) => {
   return (
     <>
       {link.path ? (
-        <Link to={link.path}>{link.label}</Link>
+        <Link to={link.path} className="button tertiary">
```

Comment:
The difference between the header "dropdown" and a regular dropdown button is that it should display on mouseover, not on click.

Reply:
The issue is that it wasn't displaying on focus. I'll make it so that it appears on mouseover AND on focus then